src/components/Register/Register.jsx

```diff
@@ -0,0 +1,142 @@
+import React, { useState } from 'react';
+import { useNavigate } from 'react-router-dom';
+import {
+  Flex,
+  FormControl,
+  FormLabel,
+  Input,
+  Button,
+  Heading,
+  Stack,
+  Box,
+  RadioGroup,
+  Radio,
+} from '@chakra-ui/react';
+import { registerWithEmailAndPassword } from '../../utils/AuthUtils';
+import styles from './Register.module.css';
+import AUTH_ROLES from '../../utils/AuthConfig';
+
+const { SUPERADMIN_ROLE, ADMIN_ROLE, DRIVER_ROLE } = AUTH_ROLES.AUTH_ROLES;
+
+const Register = () => {
+  const [firstName, setFirstName] = useState('');
+  const [lastName, setLastName] = useState('');
+  const [email, setEmail] = useState('');
+  const [phoneNumber, setPhoneNumber] = useState('');
```

Comment:
Can we add validation to the phone numbers to be valid? For now, the check can be for a length of 10 digits.